packages/edit-site/src/components/block-editor/index.js

```diff
@@ -69,6 +90,17 @@ export default function BlockEditor( { settings: _settings } ) {
 			onChange={ setContent }
 		>
 			<BlockEditorKeyboardShortcuts />
+			<Sidebar.TemplatesFill>
+				<TemplateSwitcher
```

Comment:
Can we move the template switcher to the editor header?

Reply:
Actually both features, the "add template" as well.

Reply:
I wanted to do that, but we'll need to do some work on improving dropdown menus for it. I want to integrate template parts first and get it ready feature-wise for testing.

It'd be nice to develop this in conjunction with the [new design stuff](https://github.com/WordPress/gutenberg/issues/18667). There is a lot more we can do in this screen, incrementally, that we can't do in the post editor. cc @ItsJonQ @pablohoneyhoney @jasmussen 

What do you think?

Reply:
I think that's separate, we should still be able to use a dropdown menu from the existing components and list the current template.

Reply:
Sure, but it won't look like the mockup of the template switcher that was made.

I guess, it's still a step in the right direction. I'll do it when I add template parts.